b9/include/b9/Module.hpp

```diff
@@ -26,17 +25,7 @@ struct FunctionDef {
         instructions{instructions},
         nparams{nparams},
         nlocals{nlocals} {}
-
-  // Move Constructor
-  FunctionDef(const std::string& name, std::uint32_t index,
-              std::vector<Instruction>&& instructions, std::uint32_t nparams = 0,
-              std::uint32_t nlocals = 0)
-      : name{name},
-        index{index},
-        instructions{std::move(instructions)},
-        nparams{nparams},
-        nlocals{nlocals} {}
-
+        
```

Comment:
The diff looks wierd. You added some whitespace, I think. Run clang-format?

Reply:
VS code auto indents when adding new line, and does not show up in its diff viewer by default, so I missed it. Clang-format should fix that. Good catch!